covid-dashboard/src/js/modules/map.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-undef */
+import 'leaflet/dist/leaflet-src.js';
+import 'leaflet.bigimage';
+import mapAddMarkers from './mapAddMarkers.js';
+import mapLegendInit from './mapLegendInit.js';
+import mapMenuInit from './mapMenuInit.js';
+import mapFullscreenInit from './mapFullscreenInit.js';
+import dom from './dom.js';
+
+export default () => {
+  const mymap = L.map('leafletMap', {
+    maxBounds: ([[-90, -180], [90, 180]]),
+    center: [47, 20],
+    zoom: 5,
+    zoomControl: false,
+  });
```

Comment:
There is a requirement that country should be selected when it's clicked on map.

Reply:
I can't do this requirement.

Reply:
In this case this may be replaced with clicking circle marker instead of a country..

Reply:
But now you can hover the circle and tooltip will appear.